postcss.config.js

```diff
@@ -1,7 +1,7 @@
 const autoprefixer = require('autoprefixer')
 const postCSSImport = require('postcss-import')()
 const postCSSNested = require('postcss-nested')
-const postCssCssVariables = require('postcss-css-variables')()
+const postCssCssVariables = require('postcss-css-variables')
```

Comment:
Small typo here, just to keep the pattern it's good to use `postCSS` instead of `postCss`